iphone/Classes/TiUIListView.m

```diff
@@ -2328,7 +2372,7 @@ - (void)initSearchController:(id)sender
     searchController = [[[UISearchController alloc] initWithSearchResultsController:resultViewController] retain];
     searchController.delegate = self;
     searchController.searchResultsUpdater = self;
-    searchController.hidesNavigationBarDuringPresentation = NO;
+    searchController.hidesNavigationBarDuringPresentation = YES;
```

Comment:
Please try to prevent that breaking change, I am pretty sure people will complain later.

Reply:
This is default behaviour. Though we can expose this property. Will continue to look in same.